pkgs/tools/misc/grub/2.0x.nix

```
@@ -64,6 +64,7 @@ stdenv.mkDerivation rec {
     echo 'echo "Compile grub2 with { kbdcompSupport = true; } to enable support for this command."' >> util/grub-kbdcomp.in
   '';
 
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
   nativeBuildInputs = [ bison flex python3 pkg-config autoconf automake gettext ];
```

Almost forgot to come back here with the fix.

With this, and changes in `iso-image.nix` part of WIP work not yet in a PR, I was able to successfully cross-compile a UEFI iso image.

Yes, `freetype` has to stay in both build inputs.

```suggestion
  nativeBuildInputs = [ bison flex python3 pkg-config autoconf automake gettext freetype ];
```